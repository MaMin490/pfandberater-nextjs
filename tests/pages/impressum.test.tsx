import React from 'react';
import { shallow } from 'enzyme';

import Impressum from '../../pages/impressum';

describe('Impressum', () => {
  it('should render correctly', () => {
    const component = shallow(<Impressum />);

    expect(component).toMatchSnapshot();
  });
});
