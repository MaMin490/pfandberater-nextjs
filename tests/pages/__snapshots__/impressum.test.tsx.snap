// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Impressum should render correctly 1`] = `
<div>
  Impressum
</div>
`;
